### DISCUSSION TITLE
在debian11的chrome上遇到Unable to claim interface

### DISCUSSION THREAD

**Discussion by ipv6china** [yume-chan/ya-webadb#440] [General]
在mac的浏览器上可以工作，但是在debian11的chrome上，可以看见设备，add usb设备也能工作，但是connect按钮点击后，弹出Unable to claim interface的对话框。 

我已经设置了： `chrome://flags/#unsafely-treat-insecure-origin-as-secure`这个选项。
同时也在本机运行了adb kill-server
我尝试了普通用户和root用户运行浏览器

请问还应该排查哪里？

**Comment by yume-chan** [comment]
Can you verify there is no `adb` processes after running `adb kill-server`? Maybe some programs (like Android Studio) are constantly restarting adb server.

I know some people have successfully connected to their devices on Linux so it should work.
